.vitest/src/paymaster/transport.ts

```diff
@@ -66,20 +66,24 @@ export const paymasterTransport = (
         }
       } else if (args.method === "alchemy_requestGasAndPaymasterAndData") {
         try {
-          const [{ userOperation, entryPoint, overrides }] = args.params as [
-            {
-              policyId: string;
-              entryPoint: Address;
-              dummySignature: Hex;
-              userOperation: UserOperationRequest;
-              overrides?: UserOperationOverrides;
-            },
-          ];
+          const [{ userOperation, entryPoint, dummySignature, overrides }] =
+            args.params as [
```

Comment:
Reviewer note: there's some bad type-casting happening here, because viem and aa-sdk/core's concept of a UserOperationRequest is slightly different, but so far the only issue I've needed to patch is loading the dummy signature into the UO's signature. More may come up as we increase test coverage.